components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -363,6 +363,11 @@ export class Panel extends React.Component<Props, State> {
       ? this.defaultDonationAmounts
       : donationAmounts[publisherKey]
 
+    // Ensure there is always a 0 value for custom amounts
+    if (amounts.indexOf(0) === -1) {
```

Comment:
what if we remove 0 from default list and always just add it to the beginning of the list? This could simplify things and have 0 logic only in one place

Reply:
Sure we can do that

Reply:
done